tensorflow/contrib/cmake/tf_core_framework.cmake

```diff
@@ -136,7 +204,7 @@ file(GLOB_RECURSE tf_core_framework_test_srcs
 
 list(REMOVE_ITEM tf_core_framework_srcs ${tf_core_framework_test_srcs})
 
-add_library(tf_core_framework OBJECT ${tf_core_framework_srcs})
+add_library(tf_core_framework OBJECT ${tf_core_framework_srcs} ${PROTO_TEXT_HDRS})
```

Comment:
I'm not sure of the best way to do this, but I think you need to add a dependency on `${PROTO_TEXT_SRCS}` here.